dluhc-component-library/src/components/siteSelectionForm/SiteSelectionForm.tsx

```diff
@@ -3,11 +3,13 @@ import { ReactNode } from "react";
 import { useState, useEffect, useMemo } from "preact/hooks";
 import { ValidationError } from "yup";
 import { loadJson } from "src/utils";
-import { uploadFile } from "src/api/aws/api";
 import DynamicForm from "./components/DynamicForm";
 import FormPage from "./components/FormPage";
+import CheckAnswers from "./components/CheckAnswers";
 import { FormState, FormValue, FormPageSchema, UiSchema } from "./types";
 import { createValidationSchema } from "./utils";
+import { uploadFile } from "src/api/aws/api";
```

Comment:
Our order of imports should be

Package imports
Absolute imports
Relative imports (starting with furthest away to closest)

css import